### PR TITLE
Update Templates view

### PR DESCRIPTION
**Pull request by gabaldon**
This PR updates the templates list view to match the new style.

### CODE REVIEW

File: src/components/Templates.vue [hunk 1]
```diff
@@ -1,35 +1,33 @@
...
+        >
+          <img class="add-btn" src="@/resources/svg/add.svg" />
+          <p class="text">Create a new template</p>
         </div>
         <TemplateCard
           v-for="template in paginatedItems"
           class="card"
           :name="template.name"
           :id="template.id"
+          :sources="template.radRequest.retrieve.length"
```
Comment (Tommytrg) [inline src/components/Templates.vue:28]: Could you also add `:deployedTimes`?

File: src/components/Templates.vue [hunk 2]
```diff
@@ -1,35 +1,33 @@
...
-        <div v-show="currentPage === 1" class="add">
-          <router-link to="/request/editor">
-            <img
-              data-test="create-template"
-              @click="createTemplate"
-              class="add-btn"
-              src="@/resources/svg/add.svg"
-            />
-          </router-link>
+        <div
```
Comment (Tommytrg) [inline src/components/Templates.vue:14]: All templates and create template button should have the same size

Comment (gabaldon) [reply src/components/Templates.vue:14]: They have the same size
Captura de pantalla 2020-05-07 a las 14 59 25
Captura de pantalla 2020-05-07 a las 14 59 32

File: src/components/Templates.vue (hunk 1 above)
Comment (gabaldon) [reply src/components/Templates.vue:28]: @aesedepece said this is not needed now. is it?

Comment (aesedepece) [reply src/components/Templates.vue:28]: Not yet IMO

**Pull request merged**